packages/shorebird_cli/lib/src/commands/release/release_command.dart

```diff
@@ -358,19 +358,33 @@ Use `shorebird flutter versions list` to list available versions.
       // All artifacts associated with a given release must be built
       // with the same Flutter revision.
       if (existingRelease.flutterRevision != flutterRevision) {
+        final flutterVersion = await shorebirdFlutter.getVersionForRevision(
+          flutterRevision: flutterRevision,
+        );
+
+        final formattedVersion = shorebirdFlutter.formatVersion(
```

Comment:
```suggestion
        final formattedCurrentReleaseVersion = shorebirdFlutter.formatVersion(
```

Nit: a more detailed name will help future readers of this code.